db_schema/schema.sql

```diff
@@ -43,6 +43,19 @@ CREATE TABLE spotifyEpisodeListeners (
   PRIMARY KEY (account_id, episode_id, spl_date)
 );
 
+DROP TABLE IF EXISTS spotifyAggregate;
+CREATE TABLE spotifyAggregate (
+  account_id INTEGER NOT NULL,
+  episode_id VARCHAR(128) NOT NULL,
+  spa_date DATE NOT NULL,
+  spa_age VARCHAR(128) NOT NULL,
+  spa_gender_not_specified INTEGER NOT NULL,
+  spa_gender_female INTEGER NOT NULL,
+  spa_gender_male INTEGER NOT NULL,
+  spa_gender_non_binary INTEGER NOT NULL,
+  PRIMARY KEY (account_id, episode_id, spa_date)
```

Comment:
as we have multiple rows for one episode we need spa_age in the primary key as well